api/src/main/java/org/pmiops/workbench/access/AccessUtils.java

```diff
@@ -0,0 +1,51 @@
+package org.pmiops.workbench.access;
+
+import com.google.common.collect.BiMap;
+import com.google.common.collect.ImmutableBiMap;
+import org.pmiops.workbench.actionaudit.targetproperties.BypassTimeTargetProperty;
+import org.pmiops.workbench.db.model.DbAccessModule.AccessModuleName;
+import org.pmiops.workbench.model.AccessModule;
+
+/** Utilities for RW Access related functionalities. */
+public class AccessUtils {
+  private AccessUtils() {}
+
+  private static final BiMap<AccessModule, AccessModuleName> CLIENT_TO_STORAGE_ACCESS_MODULE =
```

Comment:
We should really merge these two enums.  Is the only thing preventing this the difference between `DATA_USE_AGREEMENT` and `DATA_USER_CODE_OF_CONDUCT`?  If so, PR #5251 fixes that.

Reply:
Also COMPLIANCE_TRAINING -> RT_COMPLIANCE_TRAINING
RAS_LINK_LOGIN_GOV -> RAS_LOGIN_GOV.
Might do that in separate since that field also used in UI, and after 5251 merged...

Reply:
Yes it can certainly wait for a different PR